src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Queries/GameResultQueries.cs

```diff
@@ -222,24 +212,18 @@ private static Result MapResult(GameResultEntity gr)
         private static Penalty MapPenalty(GameResultEntity gr)
         {
             Penalty result;
-            if (gr.PenaltyTeam != 0)
-            {
-                result = new Penalty
-                {
-                    IsHomeTeam = gr.PenaltyTeam == 1,
-                    Amount = gr.PenaltyAmount,
-                    Description = gr.PenaltyDescription
-                };
-            }
-            else
+
+            result = gr.PenaltyTeam != 0 ? new Penalty
```

Comment:
tru and false branches should start on new line?

Reply:
ok